service/user/src/test/kotlin/me/helloc/techwikiplus/service/user/config/config/MySqlTestContainer.kt

```
@@ -24,19 +24,34 @@ object MySqlTestContainer {
             .withLabel("testcontainers.reuse.enable", "true")
             .apply {
                 start()
+                // JVM 종료 시 컨테이너 정리를 위한 shutdown hook 추가
+                Runtime.getRuntime().addShutdownHook(
+                    Thread {
+                        try {
+                            if (isRunning) {
+                                stop()
+                            }
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)